packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css

```diff
@@ -33,28 +33,17 @@
 }
 
 .hackaubg-landing-content p {
-    font-size: 1rem;
+    font-size: 1.2rem;
```

Comment:
I can't run it locally rn but my guess is that it's still a bit small. @SuelaCark probably should check up on that. In my opinion you should increase the font-weight as well.